src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/IndividualRepo.java

```diff
@@ -211,12 +213,12 @@ public DiscoveryNetwork network(DiscoveryNetworkDescriptor dataNetworkDescriptor
             final String dateField = dataNetworkDescriptor.getDateField();
 
             for (SolrDocument document : documents) {
+                boolean hasValidDate = false;
```

Comment:
What is the point of `hasValidDate` if it is not being used?

It gets assigned on line 219 but nothing appears to ever use it.

Reply:
Thanks. Will remove. Perhaps had plans for it and forgot.

Reply:
I recall. It was before I gave in and had the validate method also increment the year counter.